src/main/java/org/concordion/api/CommandCall.java

```diff
@@ -90,4 +91,18 @@ public String getParameter(String camelCaseParameterName, String spinalCaseParam
         }
         return null;
     }
+
+    public boolean hasNonExampleChildren() {
+        if (this.command.alwaysRunEvenIfNoNonExampleChildren()) {
+            return true;
+        }
+
+        Collection<CommandCall> children = getChildren().asCollection();
+        for (CommandCall child : children) {
+            if (!(child.getCommand().isExample())) {
+                return true;
+            }
+        }
+        return false;
+    }

```

Comment:
This was just a refactor from CommandCall (or somewhere) because it seemed to fit better here. But breaks API.